### ISSUE TITLE
intranet workflow problems

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#49]
The following problems have been identified through manipulation of test users, groups, and content on production inside the ```/permission-tests/intranet-workflow-tests``` folder.

Settings via "Sharing" tab on the folder:
--------------------------------------------------------
![perms-intra](https://cloud.githubusercontent.com/assets/263285/9429209/49005642-498e-11e5-88cb-4714c687f40b.jpg)

Placeful workflow settings:
---------------------------------------------------------------------------------------------
(via ```./placeful_workflow_configuration```)
 * For this folder: default policy
 * Below this folder: intranet

A test user was created and added to each of the indicated groups and then various content items created and manipulated. The following deviations from expectations were noted. Can we adjust as indicated?

 1. An authenticated user belonging to none of these groups, and therefore having none of the local roles was able to view content in any workflow state. By contrast, the same user, with the same lack of roles, was only able to see "published" content in ```\permission-tests\default-workflow-tests``` (this was part of the tests we conducted for #48). We would like to see similar behavior here, i.e.: authenticated users with no additional roles should be able to see only "externally published" and "internally published" content. 

 2. As was the case with #48, a user in "Test Writers Group" can submit their own content for review. They cannot submit other user's content for review because they cannot even access it (they don't have "can view"). Can we easily take away their ability to submit unless someone gives them "can edit"?

 3. As was the case with #48, a user in "Test Publishers Group" cannot check in a submitted working copy that had been checked out by someone else (although a "Test Publishers Group" user can cancel a checkout). Can we adjust to enable users in groups with "can review" to check in working copies?

**Comment by paregorios** [comment]
I have not yet finished testing and documenting for this issue, but have to step away from the task for the time being. Will add labels etc. once this issue entry is complete and is ready to be addressed.

**Comment by paregorios** [comment]
See #45 for discussion of editors being able to submit. I think there may be some prior looseness of terminology on my part contributing to confusion here.

**Comment by paregorios** [comment]
NB: child folders do not properly acquire all possible states from the intranet workflow. See separate ticket: #42.

**Comment by alecpm** [comment]
I believe the latest code (deployed on dev) addresses these issues, with the exception of the issue of visibility of "internally published" items, for which I've followed the suggestion on #42.

**Comment by paregorios** [comment]
Will test but blocked for resolution of latest problems with #40.
